pkg/auth/manager/ldap/ldap.go

```diff
@@ -191,17 +199,10 @@ func (am *mgr) Authenticate(ctx context.Context, clientID, clientSecret string)
 		Groups:      getGroupsResp.Groups,
 		Mail:        sr.Entries[0].GetEqualFoldAttributeValue(am.c.Schema.Mail),
 		DisplayName: sr.Entries[0].GetEqualFoldAttributeValue(am.c.Schema.DisplayName),
+		UidNumber:   uidNumber,
+		GidNumber:   gidNumber,
 		Opaque: &types.Opaque{
```

Comment:
Just don't initialize opaque here. It should be nil. Ditto in all the instances.

Reply:
Done